src/WebJobs.Script.WebHost/Startup.cs

```diff
@@ -15,6 +16,12 @@ public class Startup
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
+
+            if (ScriptSettingsManager.Instance.IsLinuxContainerEnvironment)
+            {
+                // Linux containers always start out in placeholder mode
+                ScriptSettingsManager.Instance.SetSetting(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
```

Comment:
Until specialization is started (via assign call) the runtime will stay in placeholder mode